griptape/tools/rest_api_client/tool.py

```diff
@@ -181,23 +203,26 @@ def get(self, params: dict) -> BaseArtifact:
                 {
                     schema.Optional(Literal("query_params", description="The request query parameters.")): dict,
                     schema.Optional(Literal("path_params", description="The request path parameters.")): list,
+                    schema.Optional(Literal("headers", description="The request headers.")): dict,
                 }
             ),
         }
     )
-    def delete(self, params: dict = None) -> BaseArtifact:
+    def delete(self, params: Optional[Dict] = None) -> BaseArtifact:
```

Comment:
I don't think the `dict = None` was correct to begin with. Let's remove that and keep it as non-`Optional`.